.github/workflows/test-e2e.yml

```diff
@@ -0,0 +1,236 @@
+
+  GO_VERSION: 1.19
+  GOFLAGS: -mod=readonly
+
+  NODE_VERSION: 16
+  CORTEZA_VERSION: 2023.9.0
+  CYPRESS_VERSION: 13.5.0
+  CYPRESS_HOST: http://localhost:8888
+  CYPRESS_BASE_URL: http://localhost:8888
+  CYPRESS_ADMIN_URL: http://localhost:8080
+  CYPRESS_COMPOSE_URL: http://localhost:8081
+  CYPRESS_WORKFLOW_URL: http://localhost:8082
+  CYPRESS_REPORTER_URL: http://localhost:8083
+  CYPRESS_ONE_URL: http://localhost:8086
+  CYPRESS_PRIVACY_URL: http://localhost:8087
+  CYPRESS_USER_EMAIL: cypress@test.com
+  CYPRESS_USER_PASSWORD: cypress123
+  CYPRESS_USER_PASSWORD_NEW: newcypress123
+  CYPRESS_USER_EMAIL_NEW: permissions@email.com
+  CYPRESS_USER_DPO: dpo@email.com
+  CYPRESS_USER_DPO_PASSWORD: dpo@email.com
+
+jobs:
+  server-client-setup:
+    name: "server and client test"
+    strategy:
+      matrix:
+        client:
+          - { port: 8081, name: compose }
+          - { port: 8080, name: admin }
+          - { port: 8082, name: workflow }
+          - { port: 8083, name: reporter }
+          - { port: 8086, name: one }
+          # - { port: 8087, name: privacy }
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
+        with:
+          limit-access-to-actor: true
+          detached: true
+
+      - name: Setup environment
+        env:
+          RELEASE_CRUST_SFTP_KEY: ${{ secrets.RELEASE_CRUST_SFTP_KEY }}
+          RELEASE_CRUST_SFTP_URI: ${{ secrets.RELEASE_CRUST_SFTP_URI }}
+          CLIENT_NAME: ${{ matrix.client.name }}
+        run: |
+          echo "SCREENSHOTS_GEN_FOLDER=$(date +"%Y%m%d%H%M%S")_$CLIENT_NAME" >> $GITHUB_ENV && \
+          echo "TMP_KEY=$(mktemp)" >> $GITHUB_ENV
+          echo $SCREENSHOTS_GEN_FOLDER
+
+      - name: Create crust key
+        env:
+          RELEASE_CRUST_SFTP_KEY: ${{ secrets.RELEASE_CRUST_SFTP_KEY }}
+          RELEASE_CRUST_KEY_FILE: ${{ env.TMP_KEY }}
+        run: |
+          echo ${RELEASE_CRUST_SFTP_KEY} | base64 -d > ${RELEASE_CRUST_KEY_FILE} && \
+          echo ${RELEASE_CRUST_SFTP_KEY} > /tmp/foo
+      - name: Make folder for uploaded screenshots
+        env:
+          RELEASE_CRUST_SFTP_URI: ${{ secrets.RELEASE_CRUST_SFTP_URI }}
+          RELEASE_CRUST_KEY_FILE: ${{ env.TMP_KEY }}
+          SCREENSHOTS_GEN_FOLDER: ${{ env.SCREENSHOTS_GEN_FOLDER }}
+        run: |
+          printf '%s\n' 'cd screenshots' "mkdir ${SCREENSHOTS_GEN_FOLDER}" | sftp -o "StrictHostKeyChecking no" -i ${RELEASE_CRUST_KEY_FILE} ${RELEASE_CRUST_SFTP_URI}
+
+      - uses: actions/setup-node@v3
+        with: { node-version: "${{ env.NODE_VERSION }}" }
+
+      - uses: actions/setup-go@v3
+        with: { go-version: "${{ env.GO_VERSION }}" }
+
+      - name: "Setup YARN"
+        run: npm install -g yarn @vue/cli-service
+
+      - name: Checkout corteza repo
+        uses: actions/checkout@v3
+        with:
+          path: corteza
+          token: ${{ secrets.GITHUB_TOKEN }}
+          ref: 2023.9.x-feature-e2e
```

Comment:
you do not need to checkout this repo as you are already in it, remove this step, but check if the folders are ok